packages/form-field/src/FormField/FormField.tsx

```diff
@@ -45,15 +46,24 @@ export const FormField = forwardRef<HTMLDivElement, FormFieldProps>(
       darkMode,
       optional,
       id,
+      'data-lgid': dataLgId = DEFAULT_LGID_ROOT,
```

Comment:
same here: do we need to provide the default here if we're setting a default in `getLgIds`